File: cpa-ios-demo/Sources/DomainsViewController.m
//
//  Copyright (c) European Broadcasting Union. All rights reserved.
//
//  License information is available from the LICENSE file.
//

#import "DomainsViewController.h"

#import "DomainViewController.h"

@implementation DomainsViewController

#pragma mark Segues

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
{
    NSAssert([segue.destinationViewController isKindOfClass:[DomainViewController class]], @"Unexpected segue destination");
    DomainViewController *domainViewController = segue.destinationViewController;
    domainViewController.domain = segue.identifier;
}

@end

File: cpa-ios-demo/Sources_Swift/DomainsViewController.swift
//
//  Copyright (c) European Broadcasting Union. All rights reserved.
//
//  License information is available from the LICENSE file.
//

import UIKit

class DomainsViewController: UITableViewController {
    
    // MARK: Segues
    
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        let domainViewController = segue.destinationViewController as! DomainViewController
        domainViewController.domain = segue.identifier
    }
}

